### ISSUE TITLE
Add 1deg structured grid for waves for SFS application 

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#2618]
### Target fix directory

wave

### Type of change

- [ ] Update existing files
- [ ] Remove files
- [X] Add new files to existing directory
- [ ] Add new fix directory

### Any workflow changes needed?

No change needed

### Related issues

While there are no changes needed to workflow to add these to the fix directories, there will be a follow-up PR with updates to use these as an option.  No version update is needed. 

The two files needed are: 
/scratch1/NCEPDEV/climate/Jessica.Meixner/SFS/wave_fix_sfs/ww3_grid.inp.glo_100
/scratch1/NCEPDEV/climate/Jessica.Meixner/SFS/wave_fix_sfs/mesh.glo_100.nc

### Pickup location

/scratch1/NCEPDEV/climate/Jessica.Meixner/SFS/wave_fix_sfs/ww3_grid.inp.glo_100 /scratch1/NCEPDEV/climate/Jessica.Meixner/SFS/wave_fix_sfs/mesh.glo_100.nc

### Size

5 MB

**Comment by JessicaMeixner-NOAA** [comment]
Code changes: https://github.com/NOAA-EMC/global-workflow/pull/2619

**Comment by WalterKolczynski-NOAA** [comment]
Files have been added to `wave/20240105` on all tier-1 machines. @KateFriedman-NOAA would you please handle AWS, Jet, and any other secondary machines?

**Comment by KateFriedman-NOAA** [comment]
Jet and AWS are updated:
```
[ =0 13:59:16 1006 role.glopara@fe6 ]
.../HFIP/hfv3gfs/glopara/git/fv3gfs> rsync -azv --delete-before role.glopara@dtn-hera.fairmont.rdhpcs.noaa.gov:/scratch1/NCEPDEV/global/glopara/fix .
receiving file list ... done
fix/wave/20240105/
fix/wave/20240105/mesh.glo_100.nc
fix/wave/20240105/ww3_grid.inp.glo_100

sent 63 bytes  received 1,149,593 bytes  28,386.57 bytes/sec
total size is 645,755,686,748  speedup is 561,694.70
```
```
[role.glopara@hfe11 AWS]$ sh copy_fix_aws_bucket.sh wave/20240105
+ aws s3 sync wave/20240105/ s3://noaa-nws-global-pds/fix/wave/20240105/
upload: wave/20240105/ww3_grid.inp.glo_100 to s3://noaa-nws-global-pds/fix/wave/20240105/ww3_grid.inp.glo_100
upload: wave/20240105/mesh.glo_100.nc to s3://noaa-nws-global-pds/fix/wave/20240105/mesh.glo_100.nc
```

**Comment by WalterKolczynski-NOAA** [comment]
Completed.
